<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="procTab">
     <attribute name="title">
      <string>进程</string>
     </attribute>
     <widget class="QTreeWidget" name="procTree">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>351</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>占用实际内存</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>占用虚拟内存</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>优先级</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>状态</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="lb_running">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>运行进程：</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_ready">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>就绪进程：</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_blocked">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>阻塞进程：</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="tb_running">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>360</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="tb_ready">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>400</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="tb_blocked">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>440</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="fileTab">
     <attribute name="title">
      <string>文件</string>
     </attribute>
     <widget class="QTreeWidget" name="fileTree">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>801</width>
        <height>441</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>文件名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>类型</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pbt_back">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>0</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbt_mkdir">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>0</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>创建目录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbt_mkfile">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>0</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>创建文件</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_path">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>461</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pbt_delete">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>0</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="memTab">
     <attribute name="title">
      <string>内存</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>471</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gl_mem"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
